metaworld/envs/mujoco/multitask_env.py

```diff
@@ -125,6 +129,19 @@ def __init__(self,
         self._active_task = 0
         self._check_env_list()
 
+    @property
```

Comment:
seems like this is no longer needed if you provide observation_space?

Reply:
right, fixed. Also added an integration test to make sure memory usage for `from_task` is as expected.